test/browser/5-request-hearing.test.ts

```diff
@@ -58,7 +58,9 @@ describe('Request a hearing', () => {
 
     /* PA11Y */
     it('checks /hearing-confirm path passes @pa11y', async () => {
-      const result = await pa11y(`${testUrl}${hearingConfirmPage.pagePath}`, pa11yOpts);
+      pa11yOpts.screenCapture = `./functional-output/hearing-confirm.png`;
```

Comment:
Do we want this screenshot to override the ones taken earlier in the test? Maybe the name should include `pa11y`?

Reply:
The previous screenshot should have been removed

Reply:
Due to the differing runtimes? I'm not sure when it archives the functional-output directory.

Reply:
I refactored so that only pa11y is taking screenshots.  I can re-implement the functional screenshots too if you want?  Can write them to separate folders